src/modules/rest/impl/src/main/java/it/geosolutions/geostore/services/rest/security/oauth2/openid/OpenIdSecurityConfiguration.java

```diff
@@ -1,11 +0,0 @@
-package it.geosolutions.geostore.services.rest.security.oauth2.openid;
```

Comment:
Why did you removed all these openId classes?

Reply:
Were a leftover and not actually used by any code involved by openid